provider/cluster/inventory.go

```diff
@@ -270,30 +280,51 @@ func (is *inventoryService) updateInventoryMetrics(inventory []ctypes.Node) {
 }
 
 func updateReservationMetrics(reservations []*reservation) {
-	inventoryReservations.WithLabelValues("quantity").Set(float64(len(reservations)))
+	inventoryReservations.WithLabelValues("none", "quantity").Set(float64(len(reservations)))
+
+	activeCPUTotal := 0.0
+	activeMemoryTotal := 0.0
+	activeStorageTotal := 0.0
+	activeEndpointsTotal := 0.0
+
+	pendingCPUTotal := 0.0
+	pendingMemoryTotal := 0.0
+	pendingStorageTotal := 0.0
+	pendingEndpointsTotal := 0.0
 
-	cpuTotal := 0.0
-	memoryTotal := 0.0
-	storageTotal := 0.0
-	endpointsTotal := 0.0
 	allocated := 0.0
 	for _, reservation := range reservations {
+		cpuTotal := &pendingCPUTotal
```

Comment:
why are pointers needed here?

Reply:
To select the destination of what set of values to update.

Reply:
ah, I see.  thanks.